src/Sarif.UnitTests/ConvertToSchemaUriTests.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
In regards to your second point, when I click Sort Usings, the usings remain in the order of Microsoft.VisualStudio.TestTools.UnitTesting first and System second. Is Visual Studio correct to maintain this order or should I manually move System to the top of the list?